bot/seasons/evergreen/bookmark.py

```diff
@@ -46,6 +63,25 @@ def __init__(self, bot: commands.Bot):
         else:
             await ctx.message.add_reaction(Emojis.envelope)
 
+        embed.add_field(
+            name=f'Bookmarked from {ctx.author.name}.',
+            value=f'[Visit original message]({ctx.message.jump_url})',
+            inline=False
+        )
+
+        def check(reaction: str, user: str) -> None:

```

Comment:
None of these type annotations are correct. Inputs to the check function match the event being waited for, [`on_reaction_add`](https://discordpy.readthedocs.io/en/latest/api.html#discord.on_reaction_add) in this case. The output is a boolean.